nix/nixos/tests/byron-proxy-validator.nix

```diff
@@ -0,0 +1,34 @@
+{ pkgs, ... }:
+
+{
+  name = "byron-proxy-validator-test";
+  nodes = {
+    proxy = { config, pkgs, ... }: {
+      imports = [
+        (import ../.)
```

Comment:
i would just do `imports = [ ../. ];`
nixos will import it for you, and give better errors when they do occur

Reply:
I agree, too much copy/paste, not enough critical thinking! Will fix and merge tonight!